assets/css/main.css

```diff
@@ -2673,17 +2681,17 @@
         }
 
             table.alt tbody tr td {
-                border: solid 1px rgba(255, 255, 255, 0.3);
+                border: solid 1px rgba(255, 255, 255, 0.0);
```

Comment:
Why not removing this line instead of having fully transparent border?

Reply:
Oh, I was experimenting with transparency and kept it in case I wanted it back. But yeah, you are right